eth/config.go

```diff
@@ -124,12 +124,12 @@ type Config struct {
 	UltraLightOnlyAnnounce bool     `toml:",omitempty"` // Whether to only announce headers, or also serve them
 
 	// Database options
-	SkipBcVersionCheck             bool `toml:"-"`
-	DatabaseHandles                int  `toml:"-"`
-	DatabaseCache                  int
-	DatabaseFreezer                string
-	DatabaseFreezerRemote          string
-	DatabaseFreezerRemoteNamespace string
+	SkipBcVersionCheck       bool `toml:"-"`
+	DatabaseHandles          int  `toml:"-"`
+	DatabaseCache            int
+	DatabaseFreezer          string
+	DatabaseFreezerRemote    string
+	DatabaseFreezerRemoteIPC bool
```

Comment:
Why bool and not a filepath string?

Reply:
This has been resolved by simply passing the raw freezer URL through to the RPC package, unmutated and unvalidated. If RPC can't handle it, it'll send the error back up and the user will see it and things will come to a grinding halt. Removes the need for the bool field.